src/IssueCreator/Main.cs

```diff
@@ -39,8 +43,20 @@ private static string CacheFolder
         }
 
 
+        private static void CurrentDomain_UnhandledException(object sender, UnhandledExceptionEventArgs e)
+        {
+            s_logger.Log((e.ExceptionObject as Exception).ToString());
```

Comment:
Probably should use `?.ToString()` in case `as` fails (if that's even possible, but if not, may as well just cast).